app/src/utils/constants.py

```diff
@@ -55,6 +55,11 @@ class LectureInternals:
     PERSON_GUEST = "1"
     PERSON_NORMAL = "2"
 
+class SponsorInternals:
+    TYPE_BRONZE = "0"
```

Comment:
I think it'll be better to use full names, i.e. `"bronze"`, `"silver"`, `"gold"` in order not to search in `SPONSOR_TYPE`. You should then change field's `max_length` accordingly to e.g. 10.